src/colours.js

```diff
@@ -0,0 +1,3 @@
+export const colours = {
```

Comment:
lets create a `tokens` folder and add this file into that - we can add others like 'spacing' etc later

Reply:
done